index.html

```diff
@@ -1,295 +1,17 @@
 <!DOCTYPE html>
```

Comment:
Automated Code Review Suggestion:

```suggestion
    <meta http-equiv="refresh" content="0; URL='src/pages/index.html'">
```